tsdb/head.go

```diff
@@ -343,6 +347,14 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 			Name: "prometheus_tsdb_mmap_chunk_corruptions_total",
 			Help: "Total number of memory-mapped chunk corruptions.",
 		}),
+		sparseHistogramsAddedTotal: prometheus.NewCounter(prometheus.CounterOpts{
+			Name: "prometheus_tsdb_sparse_histogram_added_total",
+			Help: "Total number of sparse histograms added.",
+		}),
+		sparseHistogramSeries: prometheus.NewGauge(prometheus.GaugeOpts{
+			Name: "prometheus_tsdb_sparse_histogram_series",
+			Help: "Number of sparse histogram series currently present in the memory.",
```

Comment:
I would delete the `the` and just say `Number of sparse histograms` (rather than `histogram series`, unless I understood that wrong).

Question: Is this really in memory or more "in the head block"? If it's the latter, perhaps explain it that way.